### PR TITLE
Platform local

### LINKED ISSUE #489: Travis tests previous amplifier
Currently my build is failing on travis: https://travis-ci.org/appcelerator/amp/builds/178074710

This is due to a initialization bug introduced by https://github.com/appcelerator/amp/pull/479

I'm working on a fix to the initialization bug, but the issue will still remain that `amp pf pull && amp pf start` pulls and starts amplifier from docker hub.

### PR DESCRIPTION
**Pull request by generalhenry**
closes: https://github.com/appcelerator/amp/issues/489
Adds `--local` (shorthand `-l`) option to `amp platform [pull, start, stop, status]`

Updates travis to use local amplifier image

To test:
```
make install
./shrink.sh local
amp platform pull --local
amp platform start --local
make test
amp platform status --local
amp platform stop --local
```

### CODE REVIEW

**Review by bquenin — DISMISSED** [review]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -14,8 +14,9 @@ install:
   - sudo apt-get install docker-engine
   - docker swarm init --advertise-addr 127.0.0.1
   - make install
```
Comment (bquenin) [inline .travis.yml:16]: Isn't it possible to get rid of the `make install` for travis, now that everything is containerized ?

**Comment by generalhenry** [comment]
Fix to build issue: https://github.com/appcelerator/amp/pull/492

File: .travis.yml (hunk 1 above)
Comment (generalhenry) [reply .travis.yml:16]: Yeah, we could replace the commands with
`docker run -it --rm -v /var/run/docker.sock appcelerator/amp:local amp platform pull --local`
`docker run -it --rm -v /var/run/docker.sock appcelerator/amp:local amp platform start --local`

I just think they're a bit ugly

Comment (bquenin) [reply .travis.yml:16]: You're right, it's fine like this

**Review by bquenin — APPROVED** [review]

**Pull request merged**